get-involved.php

```diff
@@ -9,7 +9,7 @@
     include $template['alert'];
 ?>
 <section class="grid hero">
-    <h1>Check out our progress for Freya Update 1</h1>
+    <h1>Check out our progress for Loki beta 1</h1>
```

Comment:
We've previously capitalized both Freya and Beta, so it would make sense for this to be too.

`Loki Beta 1`